src/CommunityToolkit.Aspire.SurrealDb/AspireSurrealDbExtensions.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire;
+using CommunityToolkit.Aspire.SurrealDb;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using SurrealDb.Net;
+
+namespace Microsoft.Extensions.Hosting;
+
+/// <summary>
+/// Provides extension methods for registering SurrealDB-related services in an <see cref="IHostApplicationBuilder"/>.
+/// </summary>
+public static class AspireSurrealDbExtensions
+{
+    private const string DefaultConfigSectionName = "Aspire:Surreal:Client";
```

Comment:
Does it make sense to use `Surreal` here and not `SurrealDb`? I'm unfamiliar with the naming conventions used within the project as to whether the `Db` suffix is expected or not.

Reply:
Same here. The suffix is not necessary here.